python/perf-kernels/gemm.py

```diff
@@ -115,7 +119,7 @@ def matmul_kernel(
         accumulator = leaky_relu(accumulator)
     # Apply scale to recover dynamic range reduced due to lower precision inputs.
     if APPLY_SCALE:
```

Comment:
It's just a question, it's not an issue with the code:
I see that fused activation function is applied before descaling. Is it correct for leaky ReLU? Furthermore, is it correct for other activation functions?

Reply:
As far as I can see the unit test doesn't cover `activation` other than "". Maybe it's a good idea to do it in the scope of another PR.

Reply:
Goo catch. Since the activation is by definition nonlinear, applying scales *before* is the right way to do it. Fixed.

Reply:
Also added TODO to test activation in another PR. Actually we want to add multiple activation functions so we can call different ones for different models.